src/EnvironmentChecker.php

```diff
@@ -103,27 +104,42 @@ public function __construct($checkSuiteName, $title)
      */
     public function init($permission = 'ADMIN')
     {
+        $canAccess = $this->canAccess(null, $permission);
+        // Allow bypassing basic-auth check if the user is logged in with the required permission
```

Comment:
The basic auth is the bypass - but I'll allow this to stay in the interest of getting this merged.